pkg/kubelet/lifecycle/handlers.go

```diff
@@ -214,7 +216,7 @@ type appArmorAdmitHandler struct {
 	apparmor.Validator
 }
 
-func (a *appArmorAdmitHandler) Admit(attrs *PodAdmitAttributes) PodAdmitResult {
+func (a *appArmorAdmitHandler) Admit(ctx context.Context, attrs *PodAdmitAttributes) PodAdmitResult {
```

Comment:
From the first look, this is a main reason of the changes in other components. Can we avoid this?

Reply:
> From the first look, this is a main reason of the changes in other components. Can we avoid this?

If we want to fully migrate, there must be a component that changes this first, and it makes sense to include this in the PR as well.

Reply:
Your observation is correct, but I'd suggest to do it when other components are migrated. Otherwise you'll have to rebase every time when those components are migrated.

Reply:
@zhifei92 JFYI: https://github.com/kubernetes/kubernetes/pull/131383

Reply:
> Otherwise you'll have to rebase every time when those components are migrated.

That makes sense, thank you.